mindsdb/integrations/handlers/sendinblue_handler/sendinblue_tables.py

```diff
@@ -1,13 +1,15 @@
 import sib_api_v3_sdk
 import pandas as pd
 
-from typing import List
-
+from typing import List, Optional, Dict, Text
+from mindsdb.utilities.log import get_log
 from mindsdb.integrations.libs.api_handler import APITable
-
+logger = get_log("integrations.sendinblue_handler")
 from mindsdb_sql.parser import ast
+from sib_api_v3_sdk.rest import ApiException
+from datetime import datetime
 
-from mindsdb.integrations.handlers.utilities.query_utilities import SELECTQueryParser, SELECTQueryExecutor
+from mindsdb.integrations.handlers.utilities.query_utilities import SELECTQueryParser, SELECTQueryExecutor, UPDATEQueryExecutor, UPDATEQueryParser, DELETEQueryParser, DELETEQueryExecutor
```

Comment:
Some of these imports seem to be unused, let's remove them. Please check all of the imports and remove any unnecessary ones.